src/snek5000_cbox/cbox.usr.f

```diff
@@ -86,6 +86,7 @@ subroutine userchk
 
       integer n, nit_pert, nit_hist
       real Pr_,Ra_
+      real vtmp(lx1*ly1*lz1*lelt,ldim),ttmp(lx1*ly1*lz1*lelt)
```

Comment:
You should assign arrays to a common block to persist arrays. You can reuse Nek5000 "scratch" common blocks as done here:

https://github.com/KTH-Nek5000/KTH_Toolbox/blob/a821507125431a091ec0ba28932440de77c2cb72/tools/stat/stat.f#L760-L781

Reply:
Done!